pkg/kubelet/pleg/generic.go

```diff
@@ -224,6 +224,15 @@ func (g *GenericPLEG) relist() {
 		}
 	}
 
+	newEventCount := 0
+	for _, value := range eventsByPodID {
+		newEventCount = newEventCount + len(value)
+	}
+	if (newEventCount + len(g.eventChannel)) > cap(g.eventChannel) {
+		klog.Errorf("new event count %d, now event channel is %d, cap is %s, channel will full, so discard",newEventCount,len(g.eventChannel),cap(g.eventChannel))
```

Comment:
We don't need to reveal the accurate number of events which will is not quite helpful to debug, so simply:

```suggestion
		klog.Errorf("event channel will be full after this relist cycle, discard",newEventCount,len(g.eventChannel),cap(g.eventChannel))
```

Reply:
OK